src/main/resources/assets/jei/lang/zh_cn.lang

```diff
@@ -17,13 +17,17 @@ jei.tooltip.recipe.id=配方 ID: %s
 jei.tooltip.not.enough.space=屏幕空间不足，无法显示 JEI 界面。
 jei.tooltip.ingredient.list.disabled=JEI 悬浮界面已禁用。
 jei.tooltip.ingredient.list.disabled.how.to.fix=按 %s 以启用。
+jei.tooltip.bookmarks=JEI 书签
+jei.tooltip.bookmarks.usage.nokey=在“按键”设置中为 JEI 书签添加按键设置。
```

Comment:
![image](https://user-images.githubusercontent.com/9113033/50580595-2d59dc80-0e05-11e9-9f6d-84bfe20550c6.png)
```suggestion
jei.tooltip.bookmarks.usage.nokey=在按键设置中为 JEI 书签添加快捷键。
```

or

```suggestion
jei.tooltip.bookmarks.usage.nokey=在“控制…”中为 JEI 书签添加按键设置。
```

I prefer the first one.